sycl/source/detail/pi.cpp

```diff
@@ -304,7 +304,8 @@ static void initializePlugins(vector_class<plugin> *Plugins) {
               << "No Plugins Found." << std::endl;
 
   PiPlugin PluginInformation{_PI_H_VERSION_STRING, _PI_H_VERSION_STRING,
-                             nullptr};
+                             nullptr, {}};
+  PluginInformation.PiFunctionTable = {};
```

Comment:
This is to correct this:
pi.cpp: error: missing field 'PiFunctionTable' initializer [-Werror,-Wmissing-field-initializers]
                             nullptr};